documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -38,12 +38,20 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
    On Manjaro or Arch Linux the package manager is ``pacman`` ::
 
     sudo pacman -S git gcc make
-    
-   For MacOS users, these packages will not come preinstalled, but can be easily obtained by installing the XCode Command Line Tools.
+
+   For MacOS users, the above packages will not come preinstalled, but can be easily obtained by installing the XCode Command Line Tools.
    These are a set of packages relevant for software development which have been bundled together by Apple. The easiest way
    to install this is to simply run one of the commands in the terminal, e.g. ``git``. The terminal will then prompt you on
    whether or not you would like to install the Command Line Tools.
 
+   For MacOS users only, download and install the latest macOS julia from here: <https://julialang.org/downloads/>. Then add julia to PATH by running:
+
+     rm -f /usr/local/bin/julia
+     ln -s /Applications/Julia-*/Contents/Resources/julia/bin/julia /usr/local/bin/julia
```

Comment:
The asterisk here got interpreted as italic. Could you add a backslash to escape the asterisk?

Reply:
I realize it looks normal in the rich diff, so it's probably fine

Reply:
Sorry, I fixed it a little bit ago, but didn't comment here.